articles/create-import-package-from-upm.md

```
@@ -0,0 +1,504 @@
+    "license": "MIT",
+    "category": "",
+    "dependencies": {}
+}
+```
+
+この状態で Unity を開くと `SampleAdder` という Package が表示されます。
+`package` というファイルを開くと、インスペクタ上で package.json の値が確認できます。
+
+![Imgur](https://i.imgur.com/QEtvyhg.png)
+
+##  Assembly Definition を作成する
```

markdownlint-cli2 (0.18.1)

Multiple spaces after hash on atx style heading

(MD019, no-multiple-space-atx)